app/assets/stylesheets/components/_modal.scss

```diff
@@ -18,24 +18,32 @@
   top: 0;
 }
 
+.modal-center {
+  display: table;
```

Comment:
In general, when dealing with a component, I'd prefer to keep all the styles that directly relate to that component in a css file. Ideally the classes would describe some piece of the component (if not the styles, explicitly). I don't know if there is room for semantic class descriptions in our atomic set-up though 😄